### ISSUE TITLE
Tail-chopping ask

### ISSUE THREAD

**Issue by johanandren** [akka/akka#29297]
I think that tail chopping will be very useful for several of the active active scenarios (hot standby, multiple active).

Currently we have an example in the https://doc.akka.io/docs/akka/current/typed/interaction-patterns.html#latency-tail-chopping but perhaps we should see if we can provide a tail chopping abstraction out of the box.

**Comment by patriknw** [comment]
Sounds like a good thing to add, maybe something specific to active-active?

Note from docs:
> There are many variations of this pattern and that is the reason this is provided as a documentation example rather than a built in Behavior in Akka. It is intended to be adjusted to your specific needs.

**Comment by johanandren** [comment]
Yep, that's my thinking, start out with active active specific in mind but see if it might be is general enough to work for any actors that you may have several of.

**Comment by johanandren** [comment]
Related: #29283
